src/models/voice_processing/samo_whisper_transcriber.py

```
@@ -0,0 +1,473 @@
+            os.makedirs(cache_dir, exist_ok=True)
+            self.model = whisper.load_model(
+                self.config.model_size, 
+                device=self.device,
+                download_root=cache_dir
+            )
+            logger.info(
+                "✅ SAMO Whisper %s model loaded successfully",
+                self.config.model_size
+            )
+        
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)